### PR TITLE
Dom update 2020-03-18

### PR DESCRIPTION
**Pull request by sandersn**
For 3.9

There are a lot of baseline changes with this one because the type of console.log changes from `(message?: any, ...optionalArguments: any[])` to `(...data: any[])`, which is more sensible anyway. I don't think the first argument is different from the rest.

### CODE REVIEW

**Comment by sandersn** [comment]
@orta reviewers list isn't working, so I'm tagging you this way in case you're interested. I'll merge after CI is green.

File: src/lib/dom.generated.d.ts [hunk 1]
```diff
@@ -2877,14 +2836,13 @@ interface CSSStyleDeclaration {
...
     counterReset: string;
-    cssFloat: string | null;
+    cssFloat: string;
     cssText: string;
     cursor: string;
     direction: string;
     display: string;
     dominantBaseline: string;
     emptyCells: string;
-    enableBackground: string | null;
```
Comment (alloy) [inline src/lib/dom.generated.d.ts]: As described [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43201#issuecomment-601731848), this is breaking the current `recharts` typings.

So heads-up to the maintainers of `recharts`: @rapmue, @royxue, @ZheyangSong, @richbai90, @caspeco-dan, @pkeuter, @jrsaunde, @crusectrl, @apalugniok, @RobertStigsson, @kousaku-maron, @iflp, @veddermatic, @g07cha, @gndelia, @dimitropoulos, @eliotball, @vkentta, @fcaylus, and @samuelweckstrom.

Comment (alloy) [reply src/lib/dom.generated.d.ts]: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43201/commits/bce5bbc752f98d07e55e5b29304de04d03a723a5

File: src/lib/dom.generated.d.ts [hunk 2]
```diff
@@ -2343,47 +2354,6 @@ declare var AudioScheduledSourceNode: {
     new(): AudioScheduledSourceNode;
 };
 
-/** A single audio track from one of the HTML media elements, <audio> or <video>.  */
```
Comment (rbuckton) [inline src/lib/dom.generated.d.ts:2346]: Curious why these disappeared, they're still part of the standard.

Comment (orta) [reply src/lib/dom.generated.d.ts:2346]: Discussion is in https://github.com/microsoft/TSJS-lib-generator/pull/802

**Pull request merged**